build_defs/languages/BUILD

```diff
@@ -0,0 +1,31 @@
+subinclude("//build_defs/sdk")
+
+proto_build_defs(
+    name = "go",
+    srcs = ["go.build_defs"],
+    visibility = ["PUBLIC"],
+    proto_languages = {
+        "grpc_language": ["go_grpc_language"],
+        "proto_language": ["go_proto_language"],
+    }
+)
```

Comment:
This creates a filegroup which are labelled appropriately such that `protoc_languages()` can generate it's languages map in the pre-build.

If this provided a custon protobuf language type, there may be additional keys in this map e.g. for `grpc_gateway`